src/wp-admin/includes/file.php

```diff
@@ -1563,6 +1563,34 @@ function wp_trusted_keys() {
 	return apply_filters( 'wp_trusted_keys', $trusted_keys );
 }
 
+/**
+ * Determines whether the given file is a valid ZIP file.
+ *
```

Comment:
I think it's worth documenting what happens if you pass in a file that doesn't exist. First pass at the text for this. 

```suggestion
 *
 * This function does not test to ensure that a file exists. Non-existent files
 * are not valid zips, so those will also return false.
 *
```

Reply:
> This function does not test to ensure that a function exists.

Should this be "not test to ensure that a file exists."?

Reply:
Updated, thanks.